src/screens/resources/index.js

```diff
@@ -0,0 +1,31 @@
+import "./resources.css";
+
+const resources = ["books", "websites", "blogs", "podcasts"];
+
+export function Resources() {
+  return (
+    <section id="resources" className="other-resources screen-content">
+      <div className="full-screen h-full w-full flex-col">
+        <div className="split-screen h-screen-50vh flex w-[35%] flex-col items-center justify-center py-16 text-center">
+          <h2>other resources you might like</h2>
+        </div>
+        <div className="split-screen h-screen-50vh relative w-full flex-row flex-wrap justify-around">
+          {resources.map((resource) => {
+            return (
+              <div className="flex w-1/2 justify-center" key={resource}>
+                <div className="flex h-full w-[75%] flex-col">
+                  {/* <div className="resource-type-heading aspect-square flex justify-center items-center">
+                      {resource}
+                    </div>
+                    <div className="resource-type-content">
+
+                    </div> */}
```

Comment:
do we need this commented code?

Reply:
This whole screen needs to be revisited. I didn't actually work on it for this pull request, other than fixing some small things. I guess Github doesn't understand a rename + some small changes.